tests/integration_tests/canal_json_adapter_compatibility/run.sh

```diff
@@ -33,7 +33,7 @@ function run() {
 	run_sql_file $CUR/data/data.sql ${UP_TIDB_HOST} ${UP_TIDB_PORT}
 
 	# sync_diff can't check non-exist table, so we check expected tables are created in downstream first
-	check_table_exists test.multi_data_type ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT} 1000
+	check_table_exists test.binary_columns ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT} 1000
```

Comment:
```suggestion
	check_table_exists test.binary_columns ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT} 200
```

Reply:
Changing to 1000 is a mistake. Let's revert it and if there are still problems we need to contact the ci team to add pod resources.